### PR TITLE
new logo, remove initial_height after loading

### PR DESCRIPTION
**Pull request by pngwn**
# Description

- Swapped the huggingface emoji for the spaces logo
- Removed the `initial_height` value when loading is complete. We could always add a `height` param that applies at all times but with no default (`initial_height` has a default) but we can see if that would be useful later.

Should make the docs look nicer when everything is loaded in:

Screenshot 2023-02-22 at 23 08 25

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have added a short summary of my change to the CHANGELOG.md
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# A note about the CHANGELOG

Hello 👋 and thank you for contributing to Gradio!

All pull requests must update the change log located in CHANGELOG.md, unless the pull request is labeled with the "no-changelog-update" label.

Please add a brief summary of the change to the Upcoming Release > Full Changelog section of the CHANGELOG.md file and include
a link to the PR (formatted in markdown) and a link to your github profile (if you like). For example, "* Added a cool new feature by `[@myusername](link-to-your-github-profile)` in `[PR 11111](https://github.com/gradio-app/gradio/pull/11111)`".

If you would like to elaborate on your change further, feel free to include a longer explanation in the other sections.
If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file using the 
GitHub web UI since that lets you upload files directly via drag-and-drop.

### CODE REVIEW

File: ui/packages/app/src/Embed.svelte [hunk 1]
```diff
@@ -7,6 +8,8 @@
 	export let space: string | null;
 	export let display: boolean;
 	export let info: boolean;
+	export let loaded: boolean;
+	$: console.log(loaded);
```
Comment (abidlabs) [inline ui/packages/app/src/Embed.svelte:12]: Remove?

**Comment by abidlabs** [comment]
Looks very nice to me! @aliabd can we try this out staging to see how it looks?

**Review by aliabd — APPROVED** [review]
Everything looks good! https://www.gradio-staging.com/docs/main

**Review by abidlabs — APPROVED** [review]
Looks great!

**Pull request merged**